api/tests/certification/session/unit/domain/usecases/get-cpf-presigned-urls_test.js

```diff
@@ -0,0 +1,107 @@
+import { expect, sinon } from '../../../../../test-helper.js';
+import { S3ObjectStorageProvider } from '../../../../../../src/shared/storage/infrastructure/providers/S3ObjectStorageProvider.js';
+import * as getPreSignedUrls from '../../../../../../src/certification/session/domain/usecases/get-cpf-presigned-urls.js';
```

Comment:
```suggestion
import { getPreSignedUrls } from '../../../../../../src/certification/session/domain/usecases/get-cpf-presigned-urls.js';
```
Est-ce que ça suffirait ?

Reply:
oui, done